kiwi/builder/live.py

```diff
@@ -241,8 +241,8 @@ def create(self):
         # calculate size and decide if we need UDF
         if rootsize.accumulate_mbyte_file_sizes() > 4096:
             log.info('ISO exceeds 4G size, using UDF filesystem')
-            custom_iso_args['create_options'].append('-allow-limited-size')
-            custom_iso_args['create_options'].append('-udf')
+            custom_iso_args['create_options'].append('-iso-level')
+            custom_iso_args['create_options'].append('3')
```

Comment:
I am not sure how to verify I am actually booting in full speed mode, however I booted without experimenting performance issues, it booted quickly (but this might be due to using a JeOS image).

On the other hand I just built an image with mkisofs using '-iso-level 3' and '-udf'  flags and I can verify it is in UDF files system, now copying it in a USB to be sure it boots in hybrid mode. Now is being build with genisoimage to test it also works with this flags.